source/_data/glossary.yml

```diff
@@ -286,15 +286,15 @@
 
 - term: Reload
   definition: >-
-    Updates Home Assistant configuration files. Changes are normally
-    automatically updated. However, when changes are made outside of the UI
-    and at a file level, Home Assistant is not aware of these changes and
-    requires the configuration file(s) to be reloaded to pick up any changes
-    made by going to **Settings** > **System** > **Restart Home Assistant**
-    (top right) > **Quick reload**. More granular reload options are available
-    in *YAML configuration reloading* section in **Developer tools** > **YAML**.
+    Refreshes Home Assistant configuration files. Changes are normally
+    automatically updated however, changes made outside of the front end
+    will not be reflected in Home Assistant and requires a reload.
+    To perform a manual reload, go to **Settings** > **System** >
+    **Restart Home Assistant** (top right) > **Quick reload**. More granular
+    reload options are available in *YAML configuration reloading* section
+    in **Developer tools** > **YAML**.
   excerpt: >
-    Updates Home Assistant configuration files. Changes are normally
+    Refreshes Home Assistant configuration files. Changes are normally
```

Comment:
```suggestion
    Applies the changes made to the Home Assistant configuration files. Changes are normally
```

Reply:
in the excerpt, after _automatically updated_, I would add the information: _However, changes made outside of the front end require a reload_. So that people who only view the tooltip get this information. Otherwise, when you only see _Changes are normally automatically updated_, I think it could be a bit confusing.